src/components/Navbar/Navbar.tsx

```diff
@@ -56,11 +59,14 @@ const Navbar = async () => {
                     size: "sm",
                   })}
                 >
-                  Login
-                  <LogInIcon className="ml-px h-5 w-5" />
+                  <span className="text-gray-900 dark:text-gray-100">
```

Comment:
The text class was likely changed mistakenly to black for light mode. Revert this change. 

Overall changes look solid, Great work, @ujjwal-207 🙌